Userland/Applications/Calendar/CalendarWidget.cpp

```diff
@@ -298,8 +299,17 @@ ErrorOr<NonnullRefPtr<GUI::Action>> CalendarWidget::create_open_settings_action(
 void CalendarWidget::create_on_tile_doubleclick()
 {
     m_event_calendar->on_tile_doubleclick = [&] {
+        for (const auto& event : m_event_calendar->event_manager().events()) {
```

Comment:
This seems like a great start, thanks! I do wonder if there's not a way that we could more strongly model the relationship between a tile and its events. Iterating over every event to find the one that matches this tile seems a bit unfortunate and slow if there's hundreds of events.

That might require a larger refactor though :)